packages/wallet-connect-sdk-core/src/index.ts

```diff
@@ -44,11 +47,17 @@ export const DEFAULT_AUTO_ACCEPT_METHODS: Method[] = ['testInvoke']
 /**
  * A list of argument types supported by wallets
  */
-export const SUPPORTED_ARG_TYPES = ['Any', 'Signature', 'Boolean', 'Integer', 'Hash160', 'Address', 'ScriptHash', 'Null', 'Hash256',
-    'ByteArray', 'PublicKey', 'String', 'ByteString', 'Array', 'Buffer', 'InteropInterface', 'Void'] as const
-/**
- * A list of networks supported by wallets
- */
+export const SUPPORTED_ARG_TYPES: ArgTypes[] = [

```

Comment:
It would be better to use from neo3-invoker

Reply:
I'm using neo3-invoker, if you look at the ArgType type you will see it. I have needed to create this const because neo3-invoker only exports a type and we do a conditional using this const.